### DISCUSSION TITLE
Support for FIDO Authentication in Meteor Accounts

### DISCUSSION THREAD

**Discussion by vlasky** [meteor/meteor#12555] [Feature Requests]
This would allow the use of a YubiKey or similar USB security key for Multifactor Authentication (MFA).

I was thinking mainly of support for U2F where the key is used alongside the username and password, however it probably is not a big stretch to support passwordless authentication using the newer FIDO2 specification as well.

**Comment by StorytellerCZ** [comment]
We discussed doing webauth here: https://github.com/meteor/meteor/discussions/11514#discussioncomment-1462542
I think that should cover most keys.

**Reply by StorytellerCZ** [reply]
Saw this today: https://oslo.js.org/
